### DISCUSSION TITLE
Unable to get project

### DISCUSSION THREAD

**Discussion by LiJie20190102** [apache/dolphinscheduler#13224] [Q&A]
Version:3.1.2
When I open the project, all projects are not displayed, but I can get them with swagger. There is no exception information in the log。
![image](https://user-images.githubusercontent.com/53458004/208381510-ff70476c-1c43-41c0-a998-61c4557c73d3.png)

![image](https://user-images.githubusercontent.com/53458004/208381817-1ec7e5d3-5b6f-4f34-9dbc-8f1889c9d44d.png)

![image](https://user-images.githubusercontent.com/53458004/208382143-5167087b-2240-4374-b212-c71ee56b9c45.png)

![image](https://user-images.githubusercontent.com/53458004/208382553-702727b7-de2c-4968-b249-29f03b96c7ba.png)

**Comment by littleziao** [comment]
Have you solved it? I encountered the same problem, but my version is 2.0.7

**Reply by LiJie20190102** [reply]
I was in a test scenario at that time. There were too many process_instance in my scenario at that time. The interface loading timed out (the timeout time was 15s). When I deleted the corresponding process_instance from the database, the interface could open normally. I think the timeout time of ds3.1.3 is still 15s

**Reply by littleziao** [reply]
Does deleting the data in t_ds_process_instance have any effect on the system? I saw below that there are 98w entries in my t_ds_process_instance table. Maybe this is the cause of the effect

**Reply by LiJie20190102** [reply]
If you delete a process_instance , the t_ds_task_instanceshould also be deleted accordingly, without any impact.
